generator/code_generator.go

```diff
@@ -1877,7 +1877,11 @@ func loadTemplate(path string) (*template.Template, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to read template: %w", err)
 	}
-	tmpl, err := template.New("").Parse(string(tmplContent))
+	tmpl, err := template.New("").Funcs(
+		map[string]any{
+			"map": CreateMap,
```

Comment:
The builtin `template` function can receive only one argument, but to pass the name of the context, we somehow need to pass multiple arguments. To circumvent the problem, I created a function to create a map.